build/Targets/References.targets

```diff
@@ -222,7 +222,7 @@
       <ItemGroup>
 
         <PackageReference Include="Moq" Version="4.2.1402.2112" />
-        <PackageReference Include="xunit" Version="2.1.0" />
+        <PackageReference Include="xunit" Version="2.2.0-beta4-build3444" />
```

Comment:
running `@powershell -NoProfile -ExecutionPolicy Bypass -Command "Measure-Command { .\build.cmd}"`

I get the following results

| xunit version | total seconds | 
| ------------- |:-------------:| 
| 2.2.0  | 86.6895754 | 
| 2.1.0  | 83.7018841 |